addon/initializers/showdown-extension.js

```diff
@@ -44,23 +58,22 @@ export function initialize(/* application */) {
         attributes[keyValue[0]] = keyValue[1];
       });
 
-      // by default we will return codeblock
-      let highlightedCodeBlock = codeblock;
+      let lineNumbersHTML = getLineNumbersHTML(idCounter, codeblock);
+      idCounter++;
 
-      assert(`Language "${language}" not found. Have you configured Prism correctly?`, Prism.languages[language])
+      assert(`Language "${language}" not found. Have you configured Prism correctly?`, Prism.languages[language]);
 
       if (language && Prism.languages[language]) {
-        highlightedCodeBlock = Prism.highlight(codeblock, Prism.languages[language], language) + end;
-        codeblock = `<pre class="language-${language}"><code ${language ? `class="${language} language-${language}"` : ''}>${highlightedCodeBlock}</code></pre>`;
+        let highlightedCodeBlock = Prism.highlight(codeblock, Prism.languages[language], language) + end;
```

Comment:
because you moved the let inside the language `if` I think the `codeblock` in the else is now a global 🤔

Reply:
It is not. Codeblock is defined at the top, highlightedCodeBlock is only used within the if statement.

Reply:
lol ignore me 🙈  I am getting myself backwards